static/scripts/dataprivacy/paginated_form.js

```diff
@@ -151,6 +161,12 @@ function setSelectionByIndex(index, event) {
 	}
 }
 
+function skipConsent() {
+	const skipConsentFor = 'student'; // TODO get from feature toggle
```

Comment:
open todo?

Reply:
Actually the function is not needed. I removed it completly.